src/configuration/configuration.ts

```diff
@@ -71,7 +71,7 @@ export function useEnterprise(): boolean {
 export function getGitHubApiUri(): string {
   if (!useEnterprise()) return DEFAULT_GITHUB_API;
   const base = getConfiguration().get<string>("github-enterprise.uri", DEFAULT_GITHUB_API).replace(/\/$/, "");
-  return base === DEFAULT_GITHUB_API ? base : `${base}/api/v3`;
+  return base === DEFAULT_GITHUB_API ? base : `api.${base}`;
```

Comment:
I _think_ this would result in the `api.` subdomain being used for GHES URLs as well. The `/api/*` path prefix is **only** allowed in GHES. 

In other words, anything in dotcom AND anything with a `.ghe.com` domain should have the `api.` subdomain. If someone is using a legitimate GHES URI, it should use the path prefix. 

Unfortunately, I think the only heuristic for this would be to look for `.ghe.com` in the URI.

Reply:
I think you're right. I'll add logic for it